src/core/plugins/download-url.js

```diff
@@ -12,6 +12,7 @@ export default function downloadUrlPlugin (toolbox) {
       const config = getConfigs()
       url = url || specSelectors.url()
       specActions.updateLoadingStatus("loading")
+      errActions.clear({type: "thrown"})
```

Comment:
A few other subsystems add `type: thrown` errors to state, can you set a `source: fetch` property and clear based on that instead?

(I know that we probably don't care about existing errors when fetching a new URL, but this may not be true in the future, or in some custom setups)